wmde/wikibase-release-pipeline
# T344549: Eliminate excessive and extraneous WDQS debug logging (fixes #417)

https://phabricator.wikimedia.org/T344549

To test this:

1) Run `./build.sh wdqs versions/wmde11.env`
2) Run `make test SUITE=repo` or any other suite, but this is a good somewhat comprehensive example
3) During test or when it completes look through `test/suites/repo/results/repo.log` and look for the `test-wdqs-*` lines to see what is now logged
